content/tutorials/token-authentication.textile

```diff
@@ -1134,11 +1147,11 @@ blang[python].
   ```
 
 blang[php].
-  Now we need to enable the login and logout functionality in the web server. In order to keep this tutorial simple, when a user logs in we'll set a cookie, and when they log out we'll clear the cookie.
+  Now you need to enable the login and logout functionality in the web server. In order to keep this tutorial simple, when a user logs in you'll set a cookie, and when they log out you'll clear the cookie.
```

Comment:
You have used 'you' here which is inconsistent with the previous versions :) This is the correct way IMO

Reply:
Changed the others to 'you'.